java/core/src/findbugs/exclude.xml

```diff
@@ -70,4 +70,8 @@
     <Class name="org.apache.orc.TypeDescription"/>
     <Method name="equals" />
   </Match>
+  <Match>
+    <Bug pattern="URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"/>
+    <Class name="org.apache.orc.impl.reader.StripePlanner$StreamInformation"/>
+  </Match>
```

Comment:
Is this relevant to this PR? Can we proceed this independently?

Reply:
Removing this, not needed.